analytics/agma/agma_module.go

```diff
@@ -0,0 +1,267 @@
+package agma
+
+import (
+	"bytes"
+	"errors"
+	"net/http"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/benbjohnson/clock"
+	"github.com/docker/go-units"
+	"github.com/golang/glog"
+	"github.com/prebid/go-gdpr/vendorconsent"
+	"github.com/prebid/prebid-server/v2/analytics"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type httpSender = func(payload []byte) error
+
+const (
+	agmaGVLID = 1122
+	p9        = 9
+)
+
+type AgamLogger struct {
```

Comment:
Curious about this. Looks like `AgamLogger` would be a typo for `AgmaLogger`, but this typo has to be repeated many times below. Typo plus cut and paste, or did you have a reason for this name?